cmd/soroban-cli/src/commands/tx/send.rs

```diff
@@ -0,0 +1,35 @@
+use soroban_rpc::GetTransactionResponse;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    XdrArgs(#[from] super::xdr::Error),
+    #[error(transparent)]
+    Config(#[from] super::super::config::Error),
+    #[error(transparent)]
+    Rpc(#[from] crate::rpc::Error),
+}
+
+#[derive(Debug, clap::Parser, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[clap(flatten)]
+    pub xdr_args: super::xdr::Args,
+    #[clap(flatten)]
+    pub config: super::super::config::Args,
+}
+
+impl Cmd {
+    pub async fn run(&self) -> Result<(), Error> {
+        let response = self.send().await?;
+        println!("{response:#?}");
```

Comment:
If we're going to output the response we should probably output it in JSON rather than the rust debug format, so it is at least usable.

I'm also wondering what's the right thing to return from here. The response makes sense I guess, it actually contains a lot of data that could be useful when scripting.